pkg/option/flags.go

```diff
@@ -182,9 +186,54 @@ func ReadAndSetFlags() error {
 		Config.ExposeStackAddresses = viper.GetBool(KeyExposeStackAddresses)
 	}
 
+	Config.CgroupRate = ParseCgroupRate(viper.GetString(KeyCgroupRate))
 	return nil
 }
 
+type CgroupRate struct {
+	Events   uint64
+	Interval uint64
+}
+
+func ParseCgroupRate(rate string) CgroupRate {
+	empty := CgroupRate{}
+
+	if rate == "" {
+		return empty
+	}
+
+	s := strings.Split(rate, ",")
+	if len(s) != 2 {
+		logger.GetLogger().Warnf("failed to parse cgroup rate '%s'", rate)
```

Comment:
if we fail to parse the cgroup string, we return an empty cgroup. From what I remember, this means that the feature is disabled (?). If so, let's add this to the message. E.g.,:
> logger.GetLogger().Warnf("failed to parse cgroup rate '%s: disabling cgroup rate throttling'", rate)

Reply:
so there's this message that we faild to parse the option and there's later another one saying that cgroup rate is disabled:

```
time="2024-04-29T07:48:30Z" level=warning msg="failed to parse cgroup rate '100,bad' : %!w(*errors.errorString=&{time: invalid duration \"bad\"})"
...
time="2024-04-29T07:48:14Z" level=info msg="Cgroup rate disabled (0/0s)"
```

and if the option is valid we display:

```
time="2024-04-29T07:49:53Z" level=info msg="Cgroup rate started (100/1s)"
```

Reply:
Sounds good, thanks!